### ISSUE TITLE
Regulation of postsynaptic potential

### ISSUE THREAD

**Issue by dosumis** [geneontology/synapse#8]
The parent class in the branch is OK, but the subclasses are confusingly named and look suspicious: 

![image](https://cloud.githubusercontent.com/assets/112839/8089303/fdec1f66-0f9e-11e5-9f3d-60ddabbeaa8f.png)

They need to be reviewed with a view to being simplified:

Current definitions:

__regulation of postsynaptic membrane potential:__  "Any process that modulates the establishment or extent of the postsynaptic membrane potential, which is generated by changes in the membrane potential of the post synaptic neuron that receives information at a synapse. The presynaptic neuron releases neurotransmitters into the synaptic cleft which bind to receptors on the postsynaptic neuron. After being bound by the neurotransmitters, these receptors can open or close an ion channel, allowing ions to enter or leave the cell and therefore altering the membrane potential of the postsynaptic neuron."

__regulation of excitatory postsynaptic membrane potential:__ "Any process that modulates the establishment or extent of the excitatory postsynaptic potential (EPSP) which is a temporary increase in postsynaptic potential due to the flow of positively charged ions into the postsynaptic cell. The flow of ions that causes an EPSP is an excitatory postsynaptic current (EPSC) and makes it easier for the neuron to fire an action potential."^^string

__negative regulation of excitatory postsynaptic membrane potential:__ Any process that prevents the establishment or decreases the extent of the excitatory postsynaptic potential (EPSP) which is a temporary increase in postsynaptic potential due to the flow of positively charged ions into the postsynaptic cell. The flow of ions that causes an EPSP is an excitatory postsynaptic current (EPSC) and makes it easier for the neuron to fire an action potential."^^string

__positive regulation of excitatory postsynaptic membrane potential:__ "Any process that enhances the establishment or increases the extent of the excitatory postsynaptic potential (EPSP) which is a temporary increase in postsynaptic potential due to the flow of positively charged ions into the postsynaptic cell. The flow of ions that causes an EPSP is an excitatory postsynaptic current (EPSC) and makes it easier for the neuron to fire an action potential."^^string

__regulation of inhibitory postsynaptic membrane potential:__ "Any process that modulates the establishment or extent of the inhibitory postsynaptic potential (IPSP) which is a temporary decrease in postsynaptic membrane potential due to the flow of negatively charged ions into the postsynaptic cell. The flow of ions that causes an IPSP is an inhibitory postsynaptic current (IPSC) and makes it more difficult for the neuron to fire an action potential."

__positive regulation of inhibitory postsynaptic membrane potential:__ "Any process that activates or increases the frequency, rate or extent of inhibitory postsynaptic potential (IPSP). IPSP is a temporary decrease in postsynaptic membrane potential due to the flow of negatively charged ions into the postsynaptic cell. The flow of ions that causes an IPSP is an inhibitory postsynaptic current (IPSC) and makes it more difficult for the neuron to fire an action potential."

(Topic started as discussion on #5 )

**Comment by dosumis** [comment]
@natashajamboree @Pimmelorus Please comment.

**Comment by dosumis** [comment]
Suggested simplification of parent class definition:

__regulation of postsynaptic membrane potential:__ "Any process that changes the membrane potential at the postsynaptic membrane, for example via the action of neutotransmistter gated ion channels as a part of synpatic transmission."

**Comment by dosumis** [comment]
Looks like we need to distinguish between regulation of membrane potential  (the y axis of this graph) and regulation of EPSP - the process depicted by this graph:

![EPSP](http://upload.wikimedia.org/wikipedia/commons/thumb/6/6b/Synapse_diag6.png/300px-Synapse_diag6.png)

**Comment by natashajamboree** [comment]
the size/shape of the EPSP will depend on the amount of neurotransmitter released, and for how long the release occurs.  
I actually think the current definition of regulation of post-synaptic membrane potential is good.

Thoughts @Pimmelorus ? Is Marcelo on this?

**Comment by Pimmelorus** [comment]
I am fine with the def of **regulation of postsynaptic membrane potential**. I am less confident on the downstream terms **regulation of excitatory postsynaptic membrane potential** and others.

My uncertainty comes from this different meanings of the terms **postsynaptic membrane potential** and **excitatory postsynaptic membrane potential**. **postsynaptic membrane potential** refers to the actual potential across the membrane in mV (the y-axis in the wikipedia image above). For instance on can say "the membrane potential is now -20 mV". An **excitatory postsynaptic membrane potential** is commonly used to indicate a __change__ in the membrane potential in the upward direction (the hump in the wikipedia image above). So, in my view a **excitatory postsynaptic membrane potential** is in itself already a subform of **regulation of postsynaptic membrane potential**. And then **regulation of the excitatory postsynaptic membrane potential** is a "regulation of regulation of the postsynaptic membrane potential"" which sounds a bit superfluous.

So, why not this:
* regulation of postsynaptic membrane potential
  * upward regulation of postsynaptic membrane potential (a.k.a. *depolarization*, *EPSP*)
  * downward regulation of postsynaptic membrane potential (a.k.a. *hyperpolarization*, *IPSP*)

Again, I am not certain about this, but I have the feeling that there are too many actual and hidden *regulation of* terms.

**Comment by dosumis** [comment]
> My uncertainty comes from this different meanings of the terms postsynaptic membrane potential and excitatory postsynaptic membrane potential. postsynaptic membrane potential refers to the actual potential across the membrane in mV (the y-axis in the wikipedia image above). For instance on can say "the membrane potential is now -20 mV". An excitatory postsynaptic membrane potential is commonly used to indicate a change in the membrane potential in the upward direction (the hump in the wikipedia image above).

Yep.  This is actually what I was trying to say in my last post.  Not so sure about your solution though.  Seems to me that, like an action potential, an EPSP is actually a process in which the membrane potential changes over time.  So, I think we should add EPSP as a process.  We can then add the specific classes for depolarization and repolarization during EPSP as part of this process.  This is basically how we model action potentials.

**Comment by mpcoba** [comment]
Hi Guys,  sorry but I don't know how's going to be the process after we have the terms defined. I assume that you are going to populate them with gene names involved in that function. So it might be more useful or good to consider excitatory and inhibitory currents and minis, EPSCs, iPSCs. Then you can discriminate between amplitude and frequency, AMPA and NMDAr currents, etc... Then this will lead you to synaptic plasticity. You will find a collection of the most studied synaptic proteins modulating these processes.postsynaptic membrane potential is to vague in my view...

**Comment by dosumis** [comment]
Hi Marcello,

We definitely want to aim for a set of terms that are useful for annotation, but we may also need to define terms for grouping and terms that break down the components of the processes we're describing.

Here's my (probably naive) start on the useful ones.

EPSP:  The process of transient depolarization and repolarisation of a postsynaptic membrane, caused by the flow of positively charged ions into the postsynaptic cell as result of opening ligand-gated ion channels.
comment: Typically assayed by measuring post-synaptic currents.
SubClassOf: 
 regulation of postsynaptic membrane potential
 part_of some synaptic transmission

mEPSP:  The process of transient depolarization and repolarisation of a postsynaptic membrane, caused by the flow of positively charged ions into the postsynaptic cell as result of opening ligand-gated ion channels as a result of spontaneous neurotransmitter release.
comment: Typically assayed by measuring post-synaptic currents.
SubClassOf: 
 EPSP
 part_of some spontaeous synaptic transmission

eEPSP: The process of transient depolarization and repolarisation of a postsynaptic membrane, caused by the flow of positively charged ions into the postsynaptic cell due to the opening ligand-gated ion channels as a result of evoked neurotransmitter release.
comment: Typically assayed by measuring post-synaptic currents.
SubClassOf: 
EPSP
part_of some evoked synaptic transmission

IPSP: The process of transient hyperpolarization of a postsynaptic membrane due to the opening of ligand-gated ion channels as a result of neurotransmitter release.
SubClassOf: 
 regulation of postsynaptic membrane potential
 part_of some synaptic transmission

This may have been what was being aimed at with the original terms (see top of ticket).  Might be interesting to look at [what is annotated to them right now](http://www.ebi.ac.uk/QuickGO/GAnnotation?a=&goid=GO%3A0060078&termUse=ancestor&relType=IPO%3D&customRelType=IPOR%2B-%3F%3D&evidence=IMP%2CIGI%2CIPI%2CIDA%2CIEP%2CEXP&protein=&tax=&qualifier=&ref=&with=&source=&q=&col=proteinDB%2CproteinID%2CproteinSymbol%2Cqualifier%2CgoID%2CgoName%2Caspect%2Cevidence%2Cref%2Cwith%2CproteinTaxon%2Cdate%2Cfrom%2Csplice&select=normal)

**Comment by Pimmelorus** [comment]
@mpcoba 
Are you in fact proposing to focus on the current rather than on its subsequent effect on the membrane potential? Makes sense from a cause-effect side.

**Comment by dosumis** [comment]
We need to be careful about disentangling assertions about involvement in a process from the evidence that something is involved in that process.  Perhaps best to treat current as evidence for 
PSP ?

**Comment by mpcoba** [comment]
well, something that I still don't know is what's going to be the level of detail that we are going to use, therefore how then the gene names are going to be annotated. You will find tons of papers describing mutant cell/animal lines, defining how hey affect AMPA, NMDAR currents, mEPSCs, not mentioning specifically postsynaptic membrane potential... So if you don't define these terms, are you going to put them under EPSP? it's ok, but it's something to decide. The same might be said to synaptic plasticity. You can just have synaptic plasticity or then LTP, LTD, and many types for each of them. Obviously these issues can go endlessly depending on who you ask. I'm thinking in the annotation process and what might also be less complicated and efficient.

**Comment by Pimmelorus** [comment]
Yes, I see that Marco has a good point by proposing that we should also create terms and relationships that are focussed on currents. 

I confirmed this by talking to Niels Cornelisse, one of the electrophysiologists in my institute. The correct train of logic is that neurotransmitters activate a ligand-gated ion channel that opens and generates a current of ions across the membrane (named EPSC and IPSC <-- note the *C* of *Current*). This in turns CAN influence the membrane potential (in the form of an EPSP or IPSP). Whether and to which extent the membrane potential is affected depends on the total net flow of charge across the membrane caused by all ion types (positively and negatively charged) at that moment.

The correct relationships then are (as far as I can see) that f.i. an AMPA receptor should be linked to the term EPSC (<-- note the *C* of *Current*). The term EPSC should be linked via "is part of ..." (?) to **EPSP**  (<-- note the *P* of *potential*) or **positive regulation of EPSP**.

But Marco, what I also understand from the communication with David is that it is not a matter of choosing one set of terms over the other, but on the description of proper relationships between concepts.

**Comment by dosumis** [comment]
> The correct train of logic is that neurotransmitters activate a ligand-gated ion channel that opens and generates a current of ions across the membrane (named EPSC and IPSC <-- note the C of Current). This in turns CAN influence the membrane potential (in the form of an EPSP or IPSP). Whether and to which extent the membrane potential is affected depends on the total net flow of charge across the membrane caused by all ion types (positively and negatively charged) at that moment.

* I'm not sure we can or should make GO  (process) terms for the current.  I think the activity terms would be sufficient for this.  It is possible to record action at the postsynaptic membrane via an annotation extension (which can also be used to record cell-type context).
  * e.g.  GO:0022848 ; acetylcholine-gated cation channel activity _that_ (__occurs\_in__ _some_ 'postsynaptic membrane') _and_ (__occurs\_in__ _some_ pyramidal cell)
* In some cases it will be useful and valid to put this activity in a wider process context: In this case the important context is the process that is the resulting PSP (a process invovling at transient change in membrane potential across the postsynaptic membrane).  We can record this using the 'involved in' pattern:
  • 'acetylcholine-gated cation channel activity invovled in eEPSP' _EquivalentTo_ 'acetylcholine-gated cation channel activity' _that_ __part\_of__ _some_ eEPSP
• It is important to remember that all GO annotation involves some inference, even when based on good experimental evidence. Typically, more inference is required for process annotations.  We just need to make sure that the inference is at least scientifically defensible and that we record the evidence.

**Comment by Pimmelorus** [comment]
@dosumis 
Ok, your explanation is clear to me.

**Comment by dosumis** [comment]
> Ok, your explanation is clear to me.
Excellent.  I can add a comment to the activity terms indicating applicability of PSC as evidence. Do you have any coments on the draft definitions for the PSP terms above.  I plan to add them soon.

**Comment by Pimmelorus** [comment]
@dosumis 
Let's wait for @mpcoba 's response. I am not the only person that is involved in the decision making. I prefer to move ahead with more members of the expert team, since I only can speak directly for my personal intuition.

**Comment by mpcoba** [comment]
I thin is ok. What it's not clear to is is, with these terms, how are you going to annotate molecules that modulate the currents?. Many papers are going to say,  KO, mutation etc.. of protein XX have a role in currents, in the way of AMPA, NMDA currents, amplitude, frequency etc... So my question is, are these going to be annotated as having a role in PSPs (which would be an inference, plus an extension) or not annotated?. In the case of being annotated, people should be taught to recognize those terms.

**Comment by dosumis** [comment]
Progress:  

Now have EPSP, IPSP and corresponding modulation terms.  Also have simply hyper and depolarization terms for postsynaptic membrane.  Further discussion is needed regarding if/when they are used in annotation as part of the synapse annotation project.

IMPORTANT: Annotations to EPSP and IPSP need to be reviewed, as likely include both core ion channels involved in EPSP/IPSP and modulators.

**Comment by RLovering** [comment]
Hi All

We appreciate that BHF-UCL and ParkinsonsUK-UCL annotations will need updating, I have downloaded the annotations we have made to these terms. I agree that we need to look at providing guidelines for the use of these terms, and also the GO annotation call next week will probably feed into this too.

It can be very useful to have a diagram available which indicates how the different components should be annotated with GO terms, for example see the first figure in http://wiki.geneontology.org/index.php/MicroRNA_GO_annotation_manual and the signaling examples in http://geneontology.org/page/go-annotation-conventions#downstream

It might be sensible to discuss this when Pim and Frank are at UCL.

Best

Ruth
